Add quiet cli-option and its implementation

zMoooooritz opened the pull request:
Implements the requested (and definitely required) feature #15 
Usage: -q
Other changes: Moved "reading configuration ..." output from stderr to stdout

florentc commented:
Thank you very much. I was waiting for user input in #15 about whether we should be quiet or verbose by default. A quiet mode seems sensible indeed.

Is there a reason to use `a+` mode instead of regular `w` mode?

We will also need to update the documentation (I can take care of it if you want).

zMoooooritz commented:
I think in this case its helpful to keep the default output without a flag.
For beginners it may not be too obvious whether they created the FIFO correctly and piped the outputs to xob in the right way, therefore a confirmation of the config and output of changes seems reasonable.

The above described could be improved in the future to ease the operation.

Indeed in this use case it is completely sufficient to use the `w` mode of freopen. I updated it.

Sorry, I have actually forgotten the docs and updated them now I hope all changes are in your sense.

And last but not least, thank you for your fast response and this awesome project

florentc commented:
Thank you. All seems well.

> I think in this case its helpful to keep the default output without a flag.
> For beginners it may not be too obvious whether they created the FIFO correctly and piped the outputs to xob in the right way, therefore a confirmation of the config and output of changes seems reasonable.

I agree. This is indeed why I made it so talkative. When using several monitors, it is also possible to misplace the bar in some dead space depending on the multihead layout. In those cases the standard output helps to confirm this is a matter of display configuration.

It seems many people use named pipes (FIFO). Ideally, you can just pipe a script like the (not that ideal) one in the README without having to bother about named pipes (which was intended to be the fallback option). Whether we should provide a default set of example input programs for xob, here or in another package, is open to discussion.
